.docforge/manifest.yaml

```diff
@@ -0,0 +1,36 @@
+nodesSelector:
+  path: /docs
+structure:
+- name: _index.md
+  properties:
+    frontmatter:
+      title: etcd Backup & Restore
+      description: A sidecar for etcd backup & restore in Gardener
```

Comment:
> A sidecar for etcd backup & restore in Gardener

Just I'm curious, Do we have to mention Gardener here as IMO etcd-backup-restore can be consumed independently as well like linode uses etcd-backup-restore.

I guess we should mention about etcd-druid
```
      description: A sidecar for etcd backup & restore deployed by etcd-operator aka etcd-druid
```
But I have no strong opinion.

Reply:
OK, changed (made Gardener only an example) @ishan16696.